### ISSUE TITLE
False positive caused by class definition.

### ISSUE THREAD

**Issue by Insti** [pwlmaciejewski/coffeelint-variable-scope#9]
Using the following code:
```
foo = ->
  class FooClass
    foovar = 1 

  bar = ->
    foovar = 2 

  console.log "foovar: #{foovar}"

foo()
```
The console log reports that `foovar` is undefined, and therefore is not in scope.
```
ReferenceError: foovar is not defined
```
However coffeelint-variable-scope reports an error:
```
⚡ #3-6: Outer scope variable overwrite. foovar.
```
